src/interfaces/index.ts

```diff
@@ -43,6 +43,13 @@ export interface Project {
   completed?: boolean;
 }
 
+export interface EthIntro {
+  path: string;
+  frontMatter: ProjectFrontMatter;
+  slug: string;
+  completed?: boolean;
+}
+
```

Comment:
If this interface is not being used there is no need to have it, right?

Maybe I'm wrong.

this interface is the same as `Project` interface and should be common for all the project's lessons